docs/_themes/clusterhq/layout_docs.html

```diff
@@ -256,10 +261,12 @@
 
             // Is version equal to this documentation release?
             if (currentversion !== DOCUMENTATION_OPTIONS.VERSION) {
+                $('link[rel="canonical"]').attr('href', 'https://docs.clusterhq.com/en/' + DOCUMENTATION_OPTIONS.VERSION + '/' + canonicalLocation);
```

Comment:
> Google respects canonical links defined by JS however do you know how it would react to this?

The experimental result from Google's tools is that it is the updated canonical link here that gets rendered by Googlebot. So, what you're suggesting is interesting because it's something I also considered - the thinking behind this way was that if in the (rare) event the AJAX call to the version indicator fails, we still have a meaningful canonical link for Google to promote `latest`, which is better than none.

However, a better approach _might_ be to assume we should canonicalize the versioned location rather than `latest` in the event that we fail to call out to `version.html`, which we could do via a failure callback on the AJAX request, as well as in the event that it succeeds and we find `x.y.z` is no longer `latest`. Not because there's any reason to think updating the canonical via JS won't be respected by Google (certainly Googlebot is aware of the update), but just to err on the side of caution and ensure we don't by failure canonicalise latest for resources that may not exist in latest or be different content.